packages/block-editor/src/components/height-control/index.js

```diff
@@ -68,6 +69,10 @@ export default function HeightControl( {
 	value,
 } ) {
 	const customRangeValue = parseFloat( value );
+	const id = useInstanceId( HeightControl, 'inspector-height-control' );
```

Comment:
In this case, I think we can change it to `block-editor-height-control` without consequences since it's a newly added ID (and I couldn't find other examples of `useInstanceId` with a prefix in the block editor package)